### PR TITLE
GDM-205: extra null check for raster layers without featureinfo

### PR DESCRIPTION
**Pull request by janvenstermans**
This can cause freeze in uploading WMS layer via wizard.

E.g. for http://grb.agiv.be/geodiensten/raadpleegdiensten/GRB-basiskaart/wms?SERVICE=WMS&version=1.3.0&request=GetCapabilities (any layer)

bug NEEDS VOTE

### CODE REVIEW

**Comment by janvenstermans** [comment]
Jan Venstermans: +1

File: gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/manager/datalayer/steps/EditLayerSettingsStep.java [hunk 1]
```diff
@@ -238,4 +237,24 @@ public void reset() {
 	public boolean stepFinished() {
 		return true;
 	}
+
+	/**
+	 * Returns whether or not the specific {@link DynamicLayerConfiguration} is of a raster type of layer.
+	 * @param layerConfig
+	 * @return
```
Comment (oliverm-be) [inline gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/manager/datalayer/steps/EditLayerSettingsStep.java]: please complete javadoc, most of the time this is trivial.

@param layerConfig the layer configuration to check
@return true if layerConfig is a raster layer

**Comment by oliverm-be** [comment]
Check my javadoc comment. 
Oliver May: +1 (binding)

**Comment by janvenstermans** [comment]
Bug fix successful:
Oliver May: +1 (binding) and solved javadoc remarks
Jan Venstermans: +1

**Pull request merged**